src/status_im/subs/wallet/send.cljs

```diff
@@ -40,14 +41,15 @@
 
 (rf/reg-sub
  :wallet/recent-recipients
- :<- [:wallet/activities-for-current-viewing-account]
+ :<- [:wallet/all-activities]
  :<- [:wallet/current-viewing-account-address]
- (fn [[sections current-viewing-account-address]]
-   (let [all-transactions        (mapcat :data sections)
-         users-sent-transactions (filter (fn [{:keys [sender]}]
-                                           (= sender current-viewing-account-address))
-                                         all-transactions)]
-     (set (map :recipient users-sent-transactions)))))
+ (fn [[all-activities current-viewing-account-address]]
+   (let [address-activity (get all-activities current-viewing-account-address)]
+     (->> address-activity
+          (keep (fn [{:keys [activity-type recipient]}]
+                  (when (= constants/wallet-activity-type-send activity-type)
+                    recipient)))
+          (distinct)))))
```

Comment:
This subscription needed the sub changed in this PR (activities for current account), so now it's adapted to return the same data but with less calculations.

Previously it executed:
 `mapcat` -> `filter` -> `map` and `set`
 
 Now: `keep` -> `distinct`